client/my-sites/stats/activity-log/index.jsx

```diff
@@ -292,32 +291,24 @@ class ActivityLog extends Component {
 		}
 	};
 
-	/**
-	 * Proceed with Restore, Backup, or Transfer since user confirmed it.
-	 * @param {string} type Type of dialog to close.
-	 */
-	handleConfirmDialog = type => {
-		const { requestedRestoreActivity, requestedBackup, siteId } = this.props;
-		let actionId = null;
+	confirmDownload = () => {
+		const { requestedBackup: { rewindId }, siteId } = this.props;
 
-		debug( `${ type } requested for after activity %o`, requestedRestoreActivity );
-		switch ( type ) {
-			case 'restore':
-				actionId = requestedRestoreActivity.rewindId;
-				this.props.rewindRestore( siteId, actionId );
-				break;
-			case 'backup':
-				actionId = requestedBackup.rewindId;
-				this.props.createBackup( siteId, actionId );
-				break;
-		}
-		scrollTo( {
-			x: 0,
-			y: 0,
-			duration: 250,
-		} );
+		this.props.createBackup( siteId, rewindId );
+		scrollTo( { x: 0, y: 0, duration: 250 } );
 	};
 
+	confirmRestore = () => {
+		const { requestedRestoreActivity: { rewindId }, siteId } = this.props;
+
+		this.props.rewindRestore( siteId, rewindId );
```

Comment:
In a recent PR where I had done something like what you're doing here, destructuring vars for a single use, you removed the destructuring and instead used `this.props.someVar`. Why is it good to do it here?
Same for `confirmDwonload`

Reply:
wasn't a strong opinion I had, but I have tended to use `this.props.actionName` on account of our eslint rule against shadowing (actually I have no problem with shadowing variable names but that's how eslint is setup)